Magento_Cms/templates/custom-country-form.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+?>
+
+<h3>Search Country</h3>
+<p id="error-message"></p>
```

Comment:
why do you need this error message if the validation message has to be working OOTB?